common/src/cams/actions.test.ts

```diff
@@ -5,14 +5,17 @@ describe('Actions', () => {
     const resource1 = {
       _actions: [Actions.ManageAssignments],
     };
+    /* eslint-disable-next-line jest/no-standalone-expect */
     expect(Actions.contains(resource1, Actions.ManageAssignments)).toBeTruthy();
```

Comment:
Interesting.  What's wrong with a stand-alone expect?

Reply:
Bad form I guess. I would like to wrap the stand alone expects in a test block.